packages/nuxt/src/module.ts

```diff
@@ -4,6 +4,11 @@ import type { ModuleHooks as NuxtI18nModuleHooks } from "@nuxtjs/i18n";
 import * as onyx from "sit-onyx";
 
 export interface ModuleOptions {
+  /**
+   * The onyx theme to use. See: [https://onyx.schwarz/development/theming.html](https://onyx.schwarz/development/theming.html)
+   * If no theme is selected the default "onyx" theme will be used.
+   */
+  theme?: "kaufland" | "lidl" | "twogo";
```

Comment:
Should we maybe maintain this type in sit-onyx itself and export it there?

Reply:
Sure, where would you like it to be?
Also: I would love to auto generate those but that would most likely require the use of macros which I'm unsure of if this is something we want inside the codebase...

Reply:
I just put it in styles/themes/index.ts for now. Let me know if you see a more fitting place ;)

Reply:
I'd suggest to move it to the types folder so we also don't need the additional entry in the package.json :)
Also maybe rename it to `OynxTheme`, what do you think?

Reply:
Sure, no problem. I just placed it close to the themes so it won't be as easy to miss when adding new ones. But I guess this shouldn't happen frequently.